src/xtgeo/grid3d/_roff_grid.py

```diff
@@ -406,6 +406,10 @@ def from_file(filelike):
             for tag_name, tag_keys in translate_kws.items()
         }
         found["filedata"] = {"filetype": None}
+
+        if isinstance(filelike, io.BytesIO):
+            filelike.seek(0)
```

Comment:
It should be the responsibility of the caller that the start of the stream points at valid data. Calling seek(0) stops the caller from managing the stream, for instance if the start of the stream does not contain roff.

Reply:
Agree but I struggled a bit with this, as setting seek(0) in the caller was not sufficient. I found now that it boils down to a weakness (when using byte streams) in the `handle_deprecated_xtgeo_roff_file` context function, and have modified that instead.